assets/queries/dockerCompose/security_opt_not_set/test/negative1.yaml

```diff
@@ -10,3 +10,5 @@ services:
       ports:
         - "8080:8080"
         - "3000:3000"
```

Comment:
```suggestion
```

Please, delete the field `ports` inside `build ` also in the positive sample.
![image](https://user-images.githubusercontent.com/74001161/160178366-010a3e9f-e3fc-4d89-9fd2-f837e139ad45.png)